MekHQ/src/mekhq/campaign/mission/AtBContract.java

```diff
@@ -345,12 +345,7 @@ public static int getEffectiveNumUnits(Campaign campaign) {
 
     public static boolean isMinorPower(String fName) {
         Faction faction = Factions.getInstance().getFaction(fName);
-        if (null != faction) {
-            return !RandomFactionGenerator.getInstance().getFactionHints().isISMajorPower(faction) &&
-                    !faction.isClan();
-        } else {
-            return false;
-        }
+        return (faction != null) && !faction.isISMajorOrSuperPower() && !faction.isClan();
```

Comment:
Honestly this seems like it belongs as a static method in the Faction class.

Reply:
This is specific to AtB Contracts, and is not equal to checking the minor power tag we have in the factions data, which is why I did not move it to factions. I think I'll move it to AtB Contract Market once I start that rework, but until then I'm going to leave it here.